### DISCUSSION TITLE
OpenTelemetry Collector Tracing not working

### DISCUSSION THREAD

**Discussion by woha** [datalust/seq-tickets#2251] [Help] [answered]
I've set up my otel-collector as described in the documentation: [Tracing from OpenTelemetry](https://docs.datalust.co/docs/tracing-from-opentelemetry-sdks).
But the otel-collector fails at startup with: `error: invalid configuration: exporters::otlp/seq: address localhost/ingest/otlp: missing port in address`.
From the Seq logs I can see that it's listening on 5341 and 45341 `Seq listening on ["http://localhost/", "https://localhost/", "http://localhost:5341/", "https://localhost:45341/"],` but when i try to configure the collector for one of those ports, it fails with: `error: invalid configuration: exporters::otlp/seq: invalid port "45341/ingest/otlp"`.

This is my otel-collector exporters config:

```exporters:
  otlp/seq:
    endpoint: https://localhost:45341/ingest/otlp
    headers:
      X-Seq-ApiKey: "..."
```

What is the correct way to set up the collector?

**Comment by KodrAus** [comment, marked as answer]
Hi @woha :wave:

It looks like the `otlp` exporter just does gRPC, which only uses the hostname and port. The path is handled by gRPC itself.

If you'd like to configure your collector to export via HTTP, you can use the `otlphttp` exporter: https://github.com/open-telemetry/opentelemetry-collector/blob/main/exporter/otlphttpexporter/README.md

**Reply by woha** [reply]
Hi @KodrAus 

and thank you very much.
This was exactly the issue. I was using the `otlp` exporter with gRPC.
I've changed it to the `otlphttp` exporter now and everything works perfectly fine.

Thank you again!
